packages/codegen/templates/app00/domains/common/components/containers/BaseLayout/BaseLayout.tsx

```diff
@@ -0,0 +1,134 @@
+/** @jsx jsx */
+import { css, jsx } from '@emotion/react'
+import styled from '@emotion/styled'
+import { Layout } from 'antd'
+import classnames from 'classnames'
+import React, { CSSProperties, FunctionComponent } from 'react'
+
+import { colors } from '@open-condo/ui/dist/colors'
+
+import { MAX_CONTENT_WIDTH } from '@condo/domains/common/constants/style'
```

Comment:
Let's try not to use `@condo` imports where possible

Reply:
move it to `codegen/generate-condo-schema.js`

Reply:
Remove all `@condo` imports from front app, instead `GlobalStyle` in `_app`.
I think we need do it in task with migration to css modules